dts/Kconfig

```diff
@@ -42,3 +42,11 @@ config HAS_DTS_SPI_DEVICE
 	help
 	  This option specifies that the target platform supports device tree
 	  configuration for spi devices, like sensors or RF modules.
+
+config HAS_DTS_USB
```

Comment:
This is currently set but never read.  Delete?

Reply:
It is used in  drivers/usb/device/Kconfig in this PR.